### PR TITLE
Cleaned up README

### PR DESCRIPTION
**Pull request by alichtman**
+ Referenced AST before defining the acronym.
+ Syntax highlighting and rephrasing where needed.
+ Grammar stuff
+ Added link to CONTRIBUTING guide ~that needs to be created~.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # saplings
 
-`saplings` is a simple library for searching, analyzing, and transforming [abstract syntax trees.](https://en.wikipedia.org/wiki/Abstract_syntax_tree) It provides some generic algorithms (saplings) that work with Python's built-in [ast](https://docs.python.org/3/library/ast.html) module. Each sapling belongs to one of two categories:
-* __Traversals:__ 
-  * Searching for nodes by type, id, attributes, or scope
-  * Generating frequency maps for specific nodes
-* __Analyses:__ 
+`saplings` is a simple library for searching, analyzing, and transforming [ASTs (abstract syntax trees.)](https://en.wikipedia.org/wiki/Abstract_syntax_tree) It provides some generic algorithms (`saplings`) that work with Python's built-in [ast](https://docs.python.org/3/library/ast.html) module. Each sapling belongs to one of two categories:
+1. __Traversals:__
+  * Searching for nodes by `type`, `id`, `attribute`, or `scope`
```
Comment (shobrook) [inline README.md]: These don't need syntax highlighting, they're not part of any syntax.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
 # saplings
 
-`saplings` is a simple library for searching, analyzing, and transforming [abstract syntax trees.](https://en.wikipedia.org/wiki/Abstract_syntax_tree) It provides some generic algorithms (saplings) that work with Python's built-in [ast](https://docs.python.org/3/library/ast.html) module. Each sapling belongs to one of two categories:
-* __Traversals:__ 
-  * Searching for nodes by type, id, attributes, or scope
-  * Generating frequency maps for specific nodes
-* __Analyses:__ 
+`saplings` is a simple library for searching, analyzing, and transforming [ASTs (abstract syntax trees.)](https://en.wikipedia.org/wiki/Abstract_syntax_tree) It provides some generic algorithms (`saplings`) that work with Python's built-in [ast](https://docs.python.org/3/library/ast.html) module. Each sapling belongs to one of two categories:
```
Comment (shobrook) [inline README.md]: This is incorrect. When you use an acronym for the first time, it should be used after the words are spelled out. See: https://english.stackexchange.com/questions/36722/how-to-use-fully-qualified-acronyms

File: README.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+1. __Traversals:__
+  * Searching for nodes by `type`, `id`, `attribute`, or `scope`
+  * Generating `frequency maps` for specific nodes
```
Comment (shobrook) [inline README.md]: Yep, again this isn't part of any programming syntax.

File: README.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 
 ## Install
 
-Compiled binaries are available for [every release,](https://github.com/shobrook/saplings/releases) and you can also install `saplings` with pip:
+Compiled binaries are available for [every release](https://github.com/shobrook/saplings/releases), and you can also install `saplings` with pip:
 
 `$ pip install saplings`
 
-Requires Python 3.0 or higher.
+Requires `Python 3.0` or higher.
```
Comment (shobrook) [inline README.md]: Not sure the highlighting is necessary.

File: README.md [hunk 5]
```diff
@@ -82,8 +82,8 @@ Documentation coming soon!
 Documentation coming soon!
 <!--- flatten() instance method-->
 
-## Adding a Sapling
+## Contributing a Sapling
```
Comment (shobrook) [inline README.md]: Sounds a little weird. Idk which one to go with.

File: README.md (hunk 4 above)
Comment (shobrook) [inline README.md]: Not sure the highlighting is necessary.

**Review by shobrook — CHANGES_REQUESTED** [review]
See comments.

File: README.md (hunk 3 above)
Comment (alichtman) [reply README.md]: I use this for emphasis, not programming syntax. You could bold it, instead, but I think this looks the best, imo.

**Review by shobrook — APPROVED** [review]

**Pull request merged**